templates/conf.d/map/header.erb

```diff
@@ -0,0 +1,11 @@
+# MANAGED BY PUPPET
```

Comment:
can you rewrite this to an epp template?

Reply:
I'd disagree with this. All other templates are ERB and I'd prefer to keep the templating language used in a module consistent. Rewriting everything to EPP is fine by me, but should happen in another PR for every template.